apps/server/src/modules/room/api/room.controller.ts

```diff
@@ -0,0 +1,38 @@
+import { ApiOperation, ApiResponse, ApiTags } from '@nestjs/swagger';
+import { Controller, ForbiddenException, Get, HttpStatus, Query } from '@nestjs/common';
+import { ApiValidationError } from '@shared/common';
+import { CurrentUser, ICurrentUser, JwtAuthentication } from '@infra/auth-guard';
+import { ErrorResponse } from '@src/core/error/dto';
+import { IFindOptions } from '@shared/domain/interface';
+import { RoomUc } from './room.uc';
+import { Room } from '../domain';
+import { RoomListResponse } from './dto/response/room-list.response';
+import { RoomMapper } from './mapper/room.mapper';
+import { RoomPaginationParams } from './dto/request/room-pagination.params';
+
+@ApiTags('Room')
+@JwtAuthentication()
+@Controller('rooms')
+export class RoomController {
+	constructor(private readonly roomUc: RoomUc) {}
+
+	@Get()
+	@ApiOperation({ summary: 'Get a list of rooms.' })
+	@ApiResponse({ status: HttpStatus.OK, description: 'Returns a list of rooms.', type: RoomListResponse })
+	@ApiResponse({ status: HttpStatus.BAD_REQUEST, type: ApiValidationError })
+	// @ApiResponse({ status: HttpStatus.UNAUTHORIZED, type: UnauthorizedException })
```

Comment:
are those for later implementation of the auth concept? otherwise having a out of the line question: whats actually the difference for the users getting an UnauthorizedException vs ForbiddenException ?